developer-docs-site/docs/tutorials/full-node/troubleshooting-fullnode.md

```diff
@@ -53,3 +53,15 @@ full_node_networks:
 [devnet_waypoint]: https://devnet.aptoslabs.com/waypoint.txt
 [aptos-labs/aptos-core]: https://github.com/aptos-labs/aptos-core.git
 [status dashboard]: https://status.devnet.aptos.dev
+
+## Incentivized Testnet Setup
+
+**Q: Generating only 1 key file instead of 3.**
+
+I am following [Run Validator Node Using Source Code](../validator-node/using-source-code.md) tutorial. One of my servers generates only 1 key file instead of 3, at step 7.
+
+**A:** Make sure you checkout the `testnet` branch as described in step 5.
```

Comment:
I reckon we're generating only a single key

it also isn't clear to me why we're mixing incentivized testnet in this

Reply:
My first point still stands, we're only generating a single file. Right?